src/rtree.ml

```diff
@@ -225,3 +225,11 @@ module Make (E : Envelope) (V : Value with type envelope = E.t) = struct
 end
 
 module Rectangle = Rectangle
+
+(* Iter Function *)
+let rec iter (rtree : Rtree.t) (f : Rtree.tree -> unit) : unit =
```

Comment:
OCaml has powerful and super useful type inference so you shouldn't need to add these type annotations (they can be useful when prototyping code).

```suggestion
let rec iter rtree f =
```

Reply:
Thanks. I have adjusted the code I encountered `unused variable iter` when building so I placed `_` to it I don't know if that is a good practice